src/main/fc/fc_tasks.c

```diff
@@ -451,11 +451,11 @@ cfTask_t cfTasks[TASK_COUNT] = {
     },
 #endif
 
-#ifdef SONAR
-    [TASK_SONAR] = {
-        .taskName = "SONAR",
-        .taskFunc = taskUpdateSonar,
-        .desiredPeriod = TASK_PERIOD_MS(50),                 // every 70 ms, approximately 20 Hz
+#ifdef RANGEFINDER
+    [TASK_RANGEFINDER] = {
+        .taskName = "RANGEFINDER",
+        .taskFunc = taskUpdateRangefinder,
+        .desiredPeriod = TASK_PERIOD_MS(50),
```

Comment:
For sonar we don't want the desired period to be shorter than the return time for the sonar pulse. For sonar with a range of 3m this is 70ms, I seem to recall. So default period for rangefinder should be 50ms. (If we add a laser or IR rangefinder then that device may want to change the period in its setup.)